src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/importering/autoimport/SykefraværsstatistikkImporteringService.java

```diff
@@ -73,15 +88,12 @@ public Importeringstatus importerHvisDetFinnesNyStatistikk() {
         log.info("Importerer ny statistikk");
         importerNyStatistikk(gjeldendeÅrstallOgKvartal);
         oppdaterPubliseringsstatus(gjeldendeÅrstallOgKvartal);
-        return Importeringstatus.IMPORTERT;
-      } else {
+      return Importeringstatus.IMPORTERT;} else {
         log.info("Statistikk er klar til importering men automatisk importering er ikke "
             + "aktivert");
-        return Importeringstatus.IKKE_AKTIVERT;
-      }
+      return Importeringstatus.IKKE_AKTIVERT;}
```

Comment:
Denne returnen og sjekken på linje 86 burde vært mye høyere opp i funksjonen... Ikke noe vits å sjekke masse data og greier for bare å kansellere